p/themes/Dark/dark.css

```diff
@@ -502,6 +502,15 @@ a.btn {
 	color: #888;
 }
 
+/*=== Scrollbar */
+.ss-scroll {
+	background: rgba(255, 255, 255, 0.3);
+}
+
+#sidebar:hover::-webkit-scrollbar-thumb, .ss-scroll {
```

Comment:
Not necessary, I just thought that It would make it a little more obvious that the split was intentional. On second thought, a comment probably would make more sense.

Reply:
Sorry, I missed your explanation higher up

Reply:
Yes indeed, I also think a very short comment would be better, also to avoid a routine optimisation.

Reply:
And it does not help to swap the two rules?

Reply:
No worries! The comment's probably the better course of action, anyways.

Reply:
Nope, tried it both ways. Firefox just stubbornly refuses to pick up on the combined line in any combination.